### ISSUE TITLE
[Feature Request] Please add std. deviation column

### ISSUE THREAD

**Issue by jpgunter** [flipperbw/FantasyPlus#26]
I would be nice to have a Std Dev column to track how close players are usually to their average vs being very boom/bust.

**Comment by brett-stayawhile** [comment]
Yes, it's definitely on my list and wouldn't be too hard as I just added Medians. I am worried that the amount of columns will (and already maybe are) quickly get out of hand though. I'm open to suggestions on how to best arrange it.

**Comment by flipperbw** [comment]
Yes, it's definitely on my list and wouldn't be too hard as I just added Medians. I am worried that the amount of columns will (and already maybe are) quickly get out of hand though. I'm open to suggestions on how to best arrange it.

**Comment by jpgunter** [comment]
Maybe add checkboxes above the table so that you can show/hide columns in real time.

**Comment by flipperbw** [comment]
That would help to hide it, but you can already do that in the settings. I'm more worried that you'll want all that data but it will be too much in your face.
